db/citations.yml

```diff
@@ -10135,6 +10135,37 @@ thamp;prb13:
   volume: '88'
   wwwemail: ''
   year: '2013'
+thatc;arxiv21:
+  author:
```

Comment:
need an `ackno` (check the key) section with the acknowledgement copy pasted there (without rogue utf8's if possible...better to copy from the source....)

Reply:
resolved